clients/py/sapphirepy/sapphire.py

```diff
@@ -41,23 +59,29 @@ def newest(self):
             return self._keys[-1]
         return None
 
-    def add(self, pk:CalldataPublicKey):
+    def add(self, pk: CalldataPublicKey):
         if self._keys:
             if self.newest['epoch'] < pk['epoch']:
                 self._keys.append(pk)
             self._trim_and_sort(pk['epoch'])
         else:
             self._keys.append(pk)
 
-def _should_intercept(method: RPCEndpoint, params:tuple[TxParams]):
+
+def _should_intercept(method: RPCEndpoint, params: tuple[TxParams]):
     if not ENCRYPT_DEPLOYS:
         if method in ('eth_sendTransaction', 'eth_estimateGas'):
             # When 'to' flag is missing, we assume it's a deployment
             if not params[0].get('to', None):
                 return False
     return method in ('eth_estimateGas', 'eth_sendTransaction', 'eth_call')
 
-def _encrypt_tx_params(pk:CalldataPublicKey, params:tuple[TxParams]):
+
+def _encrypt_tx_params(pk: CalldataPublicKey,
+                       params: tuple[TxParams],
+                       method,
+                       web3: Web3,
+                       account: Optional[LocalAccount]) -> TransactionCipher:
```

Comment:
Should provide `None` as default argument.

```python
account: Optional[LocalAccount]=None
```

Reply:
Added `None` default argument.